K8s/helm/templates/otel-collector-config.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: collector-config
+data:
+  collector-config.yaml: |
+    receivers:
+      jaeger:
```

Comment:
We should use `otlp` instead of `jaeger`. If a service is directly sending traces to a Jaeger instance then `jaeger` is fine. But in this case I feel we should use `otlp`

Reply:
The apps are currently using the jaeger trace exporter, I am keeping them as is and using the otel collector, this may be for some customer who are using jaeger.
if we want to switch to otlp we will have to change the apps as well.

Reply:
I think staying with Jaeger is fine.